documentation/docs/steps/deployIntegrationArtifact.md

```diff
@@ -0,0 +1,37 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+With this step you can deploy a integration flow artifact in to SAP Cloud Platform integration runtime using OData API.
+
+Learn more about the SAP Cloud Integration remote API for deploying an integration artifact [here](https://help.sap.com/viewer/368c481cd6954bdfa5d0435479fd4eaf/Cloud/en-US/08632076a1114bc1b6a1ecafef8f0178.html).
```

Comment:
These are not prerequisites. This should be under the description heading above, which in turn is generated from the metadata yaml file [here](https://github.com/SAP/jenkins-library/pull/2474/files#diff-c2f5178a483ee6ac8bd9dc81d16c63d1a9d2e775d82c5c8e46bd6834f37a00bfR3)